packages/vue/src/components/organisms/SfTopBar/SfTopBar.scss

```diff
@@ -0,0 +1,28 @@
+@import '../../../css/variables';
+
+.sf-topbar {
+  width: 100%;
+  height: 40px;
+
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+
+  box-sizing: border-box;
+  padding: 0px $spacer-extra-big;
+  background-color: $c-light-primary;
```

Comment:
This should be a component specific SCSS variable (top-bar-backgrounf-color), same with padding

Reply:
This has been resolved. Thanks. I didn't really understand when I read this rule. Now I do. :)